lite/examples/object_detection/android/app/src/main/java/org/tensorflow/lite/examples/detection/CameraActivity.java

```diff
@@ -156,139 +166,151 @@ public void onStateChanged(@NonNull View bottomSheet, int newState) {
           }
 
           @Override
-          public void onSlide(@NonNull View bottomSheet, float slideOffset) {}
+          public void onSlide(@NonNull View bottomSheet, float slideOffset) {
+          }
         });
 
-    frameValueTextView = findViewById(R.id.frame_info);
-    cropValueTextView = findViewById(R.id.crop_info);
-    inferenceTimeTextView = findViewById(R.id.inference_info);
-
-    apiSwitchCompat.setOnCheckedChangeListener(this);
-
-    plusImageView.setOnClickListener(this);
-    minusImageView.setOnClickListener(this);
-  }
-
-  protected int[] getRgbBytes() {
-    imageConverter.run();
-    return rgbBytes;
-  }
-
-  protected int getLuminanceStride() {
-    return yRowStride;
+    binding.bottomSheetLayout.plus.setOnClickListener(this);
+    binding.bottomSheetLayout.minus.setOnClickListener(this);
   }
 
-  protected byte[] getLuminance() {
-    return yuvBytes[0];
-  }
+  private void onStartCameraX(Size size, final int rotation) {
```

Comment:
`size` is always DESIRED_ANALYSIS_SIZE so you can use the value directly inside the method rather than passing in a parameter.

Reply:
Noted!